pywinauto/findwindows.py

```diff
@@ -239,10 +242,10 @@ def _title_match(w):
         backup_elements = elements[:]
         elements = []
         for elem in backup_elements:
-            if hasattr(elem, "_elementInfo"):
+            if hasattr(elem, "_element_info"):
                 elements.append(elem.element_info)

```

Comment:
Again. test for ```_element_info``` but append ```elem.element_info```. If this is a genuine intention the logic should be explained in comments as it's not obvious from the code